client/flutter/lib/pages/symptom_checker/question_pages/long_list_question_view.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:who_app/api/content/schema/symptom_checker_content.dart';
+import 'package:who_app/pages/symptom_checker/question_pages/previous_next_buttons.dart';
+import 'package:who_app/pages/symptom_checker/symptom_checker_model.dart';
+
+// A long list question supporting a single selection.
+class LongListQuestionView extends StatefulWidget {
```

Comment:
Definitely!  Either a superclass or a mixin.  I held off on doing this in this PR simply to avoid constraining the direction that the views would take and to try to make the PR one notch less abstract for the review...  but yep, that would be at the top of my list for the "real" views.

Reply:
Makes sense! Marking as resolved 👍